packages/core/docs/integrate/cms.md

```diff
@@ -51,36 +51,47 @@ The factory will output a composable fulfilling our interfaces. That's all you h
 
 ### Content rendering (optional)
 
-Many CMS systems allow to control the page layout by returning a list of components as a JSON file. These components are then rendered in the applicatio. The component that is mapping the `content` object into Vue components and renders them is called `RenderContent`.
+Many CMS systems allow to control the page layout by returning a list of components as a JSON file. These components are then rendered in the application. The component that is mapping the `content` object into Vue components and renders them is called `RenderContent`.
 
 ```html
-<!-- public API -->
 <RenderContent :content="content">
 ```
 
-To build this component you just have to use our `renderComponentFactory`.
+Now you need to prepare special structure for your components. You can create function, helper to filter all the metadata that you've fetched from the CMS and just leave the name and props.
```

Comment:
how about

Imho its better to ask integrator to write `extractComponents` function that has a typed interface from core (like below)